test/unit/lib/utils/telemetry/generate-payload.test.js

```diff
@@ -129,9 +129,10 @@ describe('test/unit/lib/utils/telemetry/generatePayload.test.js', () => {
     delete payload.ciName;
     expect(payload).to.have.property('commandDurationMs');
     delete payload.commandDurationMs;
+    expect(payload).to.have.property('isTtyTerminal');
```

Comment:
I've changed the approach here because these tests became flaky between local and CI offering tty or no tty and there's no easy way to enforce tty on windows Github Actions runners